app/controllers/facility_accounts_controller.rb

```diff
@@ -201,6 +191,15 @@ def show_statement
 
   private
 
+  def available_account_types
+    Account.config.account_types_for_facility(current_facility)
+  end
+
+  def current_account_type
+    return params[:account_type] if available_account_types.include?(params[:account_type])
```

Comment:
It might be clearer that you're returning a default if you don't have a valid one if you did:
```ruby
if available_account_types.include?(params[:account_type])
  params[:account_type]
else
  available_account_types.first
end
```